chpl/chpl-service/src/main/java/gov/healthit/chpl/domain/Developer.java

```diff
@@ -58,7 +58,7 @@
     /**
      * Indication of whether a health IT developer is a "self-developer" or not.
      */
-    @XmlElement(required = true)
+    @XmlElement(required = true, nillable = true)
```

Comment:
The data-model defines this as `self_developer bool NOT NULL DEFAULT false,`. Should it be nillable?

Reply:
Fixed. This was leftover from an attempt to avoid refactoring ProductDTO.